src/Workspaces/Remote/Razor/Shared/dummy.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
❗️ This file is poorly named

Reply:
sure. suggest me better name, I will use that. I just need a name.

Reply:
**WatsonReporter.cs**